src/devices/Max31865/Max31865.cs

```diff
@@ -0,0 +1,277 @@
+            WriteRead(Register.ConfigurationRead, configuration);
+
+            configuration[1] = enable ? (byte)(configuration[1] | (byte)Configuration.Bias) : (byte)(configuration[1] & ~(byte)Configuration.Bias);
+
+            configuration[0] = (byte)Register.ConfigurationWrite;
+            Write(configuration);
+        }
+
+        /// <summary>
+        /// Enable/Disable the one shot mode on the resistance temperature detector sensor
+        /// </summary>
+        private void EnableOneShot(bool enable)
+        {
+            Span<byte> configuration = stackalloc byte[2];
+            WriteRead(Register.ConfigurationRead, configuration);
+
+            configuration[1] = enable ? (byte)(configuration[1] | (byte)Configuration.OneShot) : (byte)(configuration[1] & ~(byte)Configuration.OneShot);
+
+            configuration[0] = (byte)Register.ConfigurationWrite;
+            Write(configuration);
+        }
+
+        /// <summary>
+        /// Reads the raw resistance temperature detector value and converts it to a temperature using the Callender Van Dusen temperature conversion of 15 bit ADC resistance ratio data.
+        /// </summary>
+        /// <remarks>
+        /// This math originates from: http://www.analog.com/media/en/technical-documentation/application-notes/AN709_0.pdf
+        /// </remarks>
+        /// <returns></returns>
+        private double GetTemperature()
+        {
+            short rtdNominal = (short)_prtType;
+            double z1, z2, z3, z4;
+            double temperature;
+
+            double resistance = GetResistance();
+
+            z1 = -TemperatureCoefficiantA;
+            z2 = TemperatureCoefficiantA * TemperatureCoefficiantA - (4 * TemperatureCoefficiantB);
+            z3 = (4 * TemperatureCoefficiantB) / rtdNominal;
+            z4 = 2 * TemperatureCoefficiantB;
+
+            temperature = z2 + (z3 * resistance);
+            temperature = (Math.Sqrt(temperature) + z1) / z4;
+
+            if (temperature < 0)
+            {
+                // For the following math to work, nominal resistance temperature detector resistance must be normalized to 100 ohms
+                resistance /= rtdNominal;
+                resistance *= 100;
+
+                double rpoly = resistance;
+
+                temperature = -242.02;
+                temperature += 2.2228 * rpoly;
+                rpoly *= resistance; // square
+                temperature += 2.5859e-3 * rpoly;
+                rpoly *= resistance; // ^3
+                temperature -= 4.8260e-6 * rpoly;
+                rpoly *= resistance; // ^4
+                temperature -= 2.8183e-8 * rpoly;
+                rpoly *= resistance; // ^5
+                temperature += 1.5243e-10 * rpoly;
+            }
+
+            return temperature;
+        }
+
+        /// <summary>
+        /// Read the resistance of the resistance temperature detector and return its value in Ohms.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
missing what it does return. Note that the /// comments are not mandatory for private and internal functions but if you want, you can of course use them but you still need to fill all the elements.

Reply:
Thanks, Just missed these. Will fill the return in.